fitbenchmarking/systests/test_parsers.py

```diff
@@ -0,0 +1,70 @@
+"""
+Test that accuracy of a single parser for each fitting software is
+consistent with previous versions
+"""
+
+import os
+from unittest import TestCase
+
+from fitbenchmarking.cli.main import run
+from fitbenchmarking.utils.options import Options
+
+
+class TestParsers(TestCase):
+    """
+    Test that the parsers are consistent.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Create an options file, run it, and get the results.
+        """
+
+        opts = Options()
+        opts.minimizers = {k: [v[0]] for k, v in opts.minimizers.items()}
+        opts.software = ['scipy']
```

Comment:
Do you think its worth testing all software packages here to?

Reply:
I think we discussed this in the meeting and decided to do:
all problem types -> scipy fitting
scipy problems -> all fitting softwares

I don't mind either way, although if we do test all packages here, we can basically merge these 2 test files.

Reply:
I'm happy to combine them (It's actually more thorough as it means all parsers have to work for all problem types). I think the only reason that separate parsing and minimizer tests were suggested was to save time. That being said, I doubt time will be much of a concern with a single minimizer and problem for each software and problem type respectively.